### DISCUSSION TITLE
Guide for setting up graph explore on macos

### DISCUSSION THREAD

**Discussion by satya1395** [aws/graph-explorer#79] [Q&A]
There is a guide for using Docker and EC2, but it does not include instructions for setting up Graph Explorer locally or connecting it to Neptune DB.any help ?

**Comment by cubeddu** [comment]
@satya1395 
Great question! If you're looking for instructions on setting up EC2 locally and connecting it to Neptune DB, they have a guide specifically for this purpose, which you can find [here](https://github.com/aws/graph-notebook/tree/main/additional-databases/neptune#part-1-set-up-a-ec2-proxy-server). This guide should provide you with the necessary steps to get EC2 up and running. Let me know if you need any further assistance!
